war/src/main/resources/templates/scores/details.html

```diff
@@ -108,9 +108,22 @@ <h3>Probes results</h3>
             </tr>
             </thead>
             <tbody>
-            <tr data-th-each="item: ${plugin.details}" data-th-with="probeResult = ${item.value}">
-                <td><code data-th-text="${probeResult.id}" class="label"></code></td>
-                <td data-th-text="${probeResult.message}"></td>
+             <tr data-th-each="item: ${plugin.details}" data-th-with="probeResult = ${item.value}">
+              <td><code data-th-text="${probeResult.id}" class="label"></code></td>
+              <td>
+              <td>
+                <span data-th-if="${probeResult.id == 'security'}">
+                  <span data-th-if="${probeResult.message == 'No known security vulnerabilities.'}" data-th-text="${probeResult.message}"></span>
+                  <span data-th-if="${probeResult.message != 'No known security vulnerabilities.'}">
+                       <span data-th-each="part : ${#strings.arraySplit(probeResult.message, ',')}" data-th-inline="text">
+                           <span data-th-with="parts=${#strings.arraySplit(part, ' ')}">
+                           <a data-th-text="${parts[0]}" data-th-href="${parts[1]}"></a>
+                           </span>
+                       </span>
+                  </span>
+                </span>
+                <span data-th-if="${probeResult.id != 'security'}" data-th-text="${probeResult.message}"></span>
+              </td>
```

Comment:
Indentation is not correct. 
You are adding a `span` just for the prove id condition.

Reply:
Can you please highlights the improvements @alecharp , I found 120 line number idendation is need to be changed, anything else, @alecharp

Reply:
Done @alecharp
Tested: 
<img width="1177" alt="image" src="https://github.com/jenkins-infra/plugin-health-scoring/assets/60972989/2705ac59-4206-4d82-89c7-ebccb675b8b3">